Pokemon App/Pokemon App/CardListController/CardListController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  CardListController.swift
+//  Pokemon App
+//
+//  Created by Novan Agung Waskito on 12/11/22.
+//
+
+import Foundation
+import UIKit
+
+class CardListViewController: UIViewController, UICollectionViewDelegate {
+    
+    @IBOutlet weak var cardListCollectionView: UICollectionView!
+    
+        
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationItem.searchController = searchController
+        
+        }
+        
+    }
```

Comment:
Indentationnya tolong di rapihin
```suggestion
class CardListViewController: UIViewController, UICollectionViewDelegate {
    
    @IBOutlet weak var cardListCollectionView: UICollectionView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        
        navigationItem.searchController = searchController
    }
        
}
```

Reply:
@jaygung17 ini tolong benerin juga